packages/tupaia-web/src/features/Map/utils/processMeasureData.ts

```diff
@@ -13,24 +13,32 @@ import {
 import { Entity } from '@tupaia/types';
 
 interface processMeasureDataProps {
+  activeEntityCode?: string;
```

Comment:
Did we end up using this `activeEntityCode` param? Looks like no?

Reply:
No we didn't 👍